### ISSUE TITLE
Could you pease give an example code about pullquote

### ISSUE THREAD

**Issue by xiejiachen** [LouisBarranqueiro/hexo-theme-tranquilpeak#179]
I have read your response to another asker with your demo web.

And I tied that code you give, but works not good
 Could you please give an example codes with some word？

**Comment by LouisBarranqueiro** [comment]
Here is an example : 
``` 
{% pullquote right %}
Praesent diam elit, interdum ut pulvinar placerat, imperdiet at magna.
{% endpullquote %}
```
I just noticed that there is an unwanted margin top in the pullquote, I will fix that in v1.5.1.

**Comment by LouisBarranqueiro** [comment]
[Tranquilpeak v1.6.0](https://github.com/LouisBarranqueiro/hexo-theme-tranquilpeak/releases/tag/v1.6.0) is released and solve this issue
